### PR TITLE
Mongo db Atlass connection

### PR DESCRIPTION
**Pull request by lutitech**

### CODE REVIEW

File: src/bin/www [hunk 1]
```diff
@@ -12,7 +12,7 @@ var http = require('http');
  * Get port from environment and store in Express.
  */
 
-var port = normalizePort(process.env.PORT || '3000');
+var port = normalizePort(process.env.PORT || '5000');
```
Comment (dejimania) [inline src/bin/www]: kindly change var to const or let. Regards

**Review by Busry — DISMISSED** [review]
good one

**Review by dejimania — DISMISSED** [review]

**Comment by dinorhythms** [comment]
LGTM

**Review by dinorhythms — DISMISSED** [review]

**Review by dinorhythms — APPROVED** [review]

**Review by dejimania — DISMISSED** [review]

**Review by dinorhythms — APPROVED** [review]

**Review by Busry — DISMISSED** [review]
nice guys

**Review by dinorhythms — APPROVED** [review]
LGTM

**Review by dinorhythms — DISMISSED** [review]

**Review by Busry — APPROVED** [review]
good

**Review by dejimania — APPROVED** [review]

**Pull request merged**